settings/R/create.multisite.settings.r

```diff
@@ -0,0 +1,55 @@
+##' @export
+createMultiSiteSettings <- function(template.settings, site.ids, out.dir) {
+  if(missing(out.dir)) {
+    if(is.null(template.settings$outdir)) {
+      warning("No out.dir set!")
+      out.dir <- NULL
+    } else {
+      out.dir <- template.settings$outdir
+    }
+  }
+  
+  template.settings <- .set.out.dir(template.settings, out.dir)
+  
+  run.settings <- lapply(site.ids, .get.run.settings, template.settings=template.settings)
+  
+  template.settings[["run", global=FALSE]] <- run.settings
+  return(template.settings)
+}
+
+##' @export
+.get.run.settings <- function(template.settings, site.id) {
+  start.date = template.settings$run$start.date
```

Comment:
OK, but honestly maybe it's better if I just move this out of the package. I'm really designing it to be specific to my use case, not necessarily generically (e.g., the assumption that `met.start` and `met.end` should be the same as `start.date` and `end.date`. 

For the general case (to the extent there is one), I think it's a bad idea for `createMultiSiteSettings()` to depend on what's in `run.settings`. Don't you think it should just take `template.settings$run` and replace `site.id` appropriately? Anything more complex than that falls into an "advanced" use case in my mind, for which users should be deciding for themselves what the `run` blocks should look like.